tests/apollo/CMakeLists.txt

```diff
@@ -1,4 +1,23 @@
-set(APOLLO_TEST_ENV "BUILD_ROCKSDB_STORAGE=${BUILD_ROCKSDB_STORAGE}")
+set(APOLLO_TEST_ENV "BUILD_ROCKSDB_STORAGE=${BUILD_ROCKSDB_STORAGE} USE_S3_OBJECT_STORE=${USE_S3_OBJECT_STORE}")
```

Comment:
shouldn't you also pass the `USE_S3_OBJECT_STORE` parameter via travis.yml, so that it runs as part of CI ?

Reply:
You have a point here. We can do this, but if USE_S3_OBJECT_STORE will be the default behaviour, why not change the build itself to always use it?

Reply:
As discussed offline, let's wire this via `.travis.yml`, with USE_S3_OBJECT_STORE always active.

Hint: you can use the `env.global` section to achieve this.